pkgs/by-name/si/simplemoji/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  autoPatchelfHook,
+  fetchFromGitHub,
+  lib,
+  libxkbcommon,
+  fontconfig,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+  wayland,
+  xorg,
+  noto-fonts-color-emoji,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "simplemoji";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "SergioRibera";
+    repo = pname;
```

Comment:
```suggestion
    repo = "simplemoji";
```

Reply:
Real question, why not use the `pname` recursively?

Reply:
`pname` and the repository name are not inherently connected. While they happen to be the same in this case, one is not a function of the other.

Reply:
https://github.com/NixOS/nixpkgs/issues/277994

Reply:
@h7x4 @AndersonTorres Amazing, thanks a lot for the explanation, here I fixed it